packages/perseus-core/src/utils/split-perseus-item.ts

```diff
@@ -3,18 +3,18 @@ import _ from "underscore";
 import {getPublicWidgetOptionsFunction} from "../widgets/core-widget-registry";
 import {getUpgradedWidgetOptions} from "../widgets/upgrade";
 
-import type {PerseusItem} from "../data-schema";
+import type {PerseusItem, PerseusRenderer} from "../data-schema";
 
 /**
- * Return a copy of a Perseus item with rubric data removed (ie answers)
+ * Return a copy of a PerseusRenderer with rubric data removed (ie answers)
  *
- * @param originalItem - the original, full Perseus item (which includes the rubric - aka answer data)
+ * @param original - the original, full PerseusRenderer (which includes the rubric - aka answer data)
  */
-export default function splitPerseusItem(
-    originalItem: PerseusItem,
-): PerseusItem {
-    const item = _.clone(originalItem);
-    const originalWidgets = item.question.widgets ?? {};
+export function splitPerseusRenderer(
+    original: PerseusRenderer,
+): PerseusRenderer {
+    const clone = _.clone(original);
```

Comment:
It's not clear to me why we need to clone here, but I don't think we need to address that in this PR.

Reply:
Just overly cautious on my part trying to avoid mutating incoming objects.